nsfc-plugin/src/main/java/org/osc/controller/nsfc/api/NeutronSfcSdnControllerApi.java

```diff
@@ -54,27 +44,11 @@
                  SUPPORT_NEUTRON_SFC + ":Boolean=true"})
 public class NeutronSfcSdnControllerApi implements SdnControllerApi {
 
-    @Reference(target = "(osgi.local.enabled=true)")
-    private TransactionControl txControl;
-
-    @Reference(target = "(osgi.unit.name=nsfc-mgr)")
-    private EntityManagerFactoryBuilder builder;
-
-    @Reference(target = "(osgi.jdbc.driver.class=org.h2.Driver)")
-    private DataSourceFactory jdbcFactory;
-
-    @Reference(target = "(osgi.local.enabled=true)")
-    private JPAEntityManagerProviderFactory resourceFactory;
-
-    private static final Logger LOG = LoggerFactory.getLogger(NeutronSfcSdnControllerApi.class);
-
     private static final String VERSION = "0.1";
     private static final String NAME = "Neutron-sfc";
 
-    private static final String DB_URL_PREFIX = "jdbc:h2:./nsfcPlugin_";
-    private static final String DB_USER = "admin";
-    private static final String DB_PASSWORD = "admin123";
-    private static final String URL_OPTS = ";MVCC\\=TRUE;LOCK_TIMEOUT\\=10000;MV_STORE=FALSE;";
+    private static final String AUTH_URL_LOCAL = "/v3";
```

Comment:
please add the comment when we get official release of openstack4j, this URL might change?

Reply:
I thought it won't change as long as we are using version 3 of the API.